schema/4.0.0/schema.md

```diff
@@ -1112,6 +1112,71 @@ See also `default_embedding` in `uns`.
 </tbody></table>
 <br>
 
+
+### {category}_colors
+
+<table><tbody>
+  <tr>
+    <th>Key</th>
+      <td>
+        {category}_colors where {category} MUST be the name of a <code>category</code> data type column in <code>obs</code> that<br> is annotated by the data submitter or curator. The following columns that are annotated by CELLxGENE<br> Discover MUST NOT be specified as {category}:<br><br>
+      <ul>
+        <li>assay</li>
+        <li>cell_type</li>
+        <li>development_stage</li>
+        <li>disease</li>
+        <li>organism</li>
+        <li>self_reported_ethnicity</li>
+        <li>sex</li>
+        <li>tissue</li>       
```

Comment:
I feel like I'm left hanging with a "but what if I want to define the palette for one of those fields?"
After the list add "Instead, annotate {category}_ontology_term_id_colors for those columns"?

Reply:
Addressed in b1a41adbfffc1a5ee1280deeb13fec4460abad8a.

Reply:
@jahilton - I could also create a table if that would be easier to comprehend:

| **To color** | **Use**                    |
|--------------|----------------------------|
| assay        | assay_ontology_term_id     |
| cell_type    | cell_type_ontology_term_id |

Reply:
I think the added line is sufficient